cea/technologies/storage_tank_pcm.py

```diff
@@ -225,23 +231,23 @@ def charge_storage(self, load_to_storage_Wh):
                     effective_load_to_storage_Wh = self.size_Wh - state_the_storage_after_thermal_gain
                     new_storage_capacity_Wh = self.size_Wh
                     new_phase = self.new_phase_tank(new_storage_capacity_Wh)
-                    new_T_tank_K = self.new_temperature_tank(new_phase, new_storage_capacity_Wh, -effective_load_to_storage_Wh)
+                    new_T_tank_K = self.new_temperature_tank(new_phase, new_storage_capacity_Wh, self.current_thermal_gain_Wh-effective_load_to_storage_Wh)
```

Comment:
new_T_tank_K = self.new_temperature_tank(new_phase, new_storage_capacity_Wh, self.current_thermal_gain_Wh - effective_load_to_storage_Wh)

Reply:
added spacing between math signals

Reply:
changed. (also on lines 220 and 225)